meta-ostro/conf/distro/ostro.conf

```diff
@@ -236,6 +236,9 @@ PREFERRED_PROVIDER_virtual/java-initial-native = "cacao-initial-native"
 PREFERRED_PROVIDER_virtual/java-native = "cacao-native"
 PREFERRED_PROVIDER_virtual/javac-native = "ecj-bootstrap-native"
 
+# Java runtime
+PREFERRED_RPROVIDER_java2-runtime = "openjre-8"
```

Comment:
Well, for Java *development* that makes sense. java2-runtime is just used for running Java programs. Does the jdk re-use the jre (i.e. depends on it) or is it something else entirely?

If it depends on it, then java2-runtime = "openjre-8" is fine: we use only that when packagegroup-java-jdk.bb isn't installed, and we add the extra pieces from the jdk when the package group is installed.